Tasks/AutomatedAnalysisV0/task.json

```diff
@@ -0,0 +1,468 @@
+{
+    "id": "6B519857-1A20-4248-BAE8-AAD039015AFB",
+    "name": "AutomatedAnalysis",
+    "friendlyName": "Automated Analysis",
+    "description": "Do a automated analysis of your appication metrics ",
+    "author": "Microsoft Corporation",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/azure-monitor",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=870240)",
+    "category": "Utility",
+    "releaseNotes": "What's new in Version 1.0:<br/>&nbsp;Added support for query unified Azure monitor alerts.",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Server"
+    ],
+    "version": {
+        "Major": 1,
+        "Minor": 163,
+        "Patch": 1
+    },
+    "preview": true,
+    "instanceNameFormat": "Automated Analysis",
+    "groups": [
+        {
+            "name": "providerInputs",
+            "displayName": "Provider inputs",
+            "isExpanded": false
+        },
+        {
+            "name": "metricInputs",
+            "displayName": "Metric inputs",
+            "isExpanded": false
+        },
+        {
+            "name": "orchestrationInputs",
+            "displayName": "Orchestration inputs",
+            "isExpanded": false
+        },
+        {
+            "name": "preprocessingInputs",
+            "displayName": "Preprocessing inputs",
+            "isExpanded": false
+        },
+        {
+            "name": "classificationInputs",
+            "displayName": "Classification inputs",
+            "isExpanded": false
+        },
+        {
+            "name": "scoringInputs",
+            "displayName": "Scoring inputs",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "connectedServiceNameARM",
+            "type": "connectedService:AzureRM",
+            "label": "Azure subscription",
+            "defaultValue": "",
+            "required": "true",
+            "helpMarkDown": "Select an Azure Resource Manager subscription"
+        },
+        {
+            "name": "ResourceGroupName",
+            "type": "pickList",
+            "label": "Resource group",
+            "required": true,
+            "properties": {
+                "EditableOptions": "True"
+            },
+            "helpMarkDown": "Provide the name of a resource group"
+        },
+        {
+            "name": "containerGroupName",
+            "type": "string",
+            "label": "Container group name",
+            "required": true,
+            "helpMarkDown": "Name of the container group to create to run the analysis module. Specified container group will be created if it does not exist "
```

Comment:
"Name of the container group to run the analysis module. Specified container group will be created if it does not exist."